src/lib/api-client.js

```diff
@@ -115,3 +115,11 @@ export function searchStudent(id){
         Dispatcher.handleAction('NO_STUDENT_FOUND');
     });
 }
+
+export function getAllModels(){
+    get('model/all', {}
+    ).then(data => {
+        Dispatcher.handleAction('MODELS_RECEIVED', data);
+        hashHistory.push('/models');
```

Comment:
I did this because it seemed like the right place to put it based on other searchStudent method in the same file. Idk where else to put it, so let me know.